.github/workflows/release.yml

```diff
@@ -0,0 +1,100 @@
+name: trino-python-client release
+
+on:
+  workflow_dispatch:
+
+env:
+  PYTHON_VERSION: "3.9.5"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{env.PYTHON_VERSION}}
+
+      - name: Test release
+        run: |
+          python3 -m venv .venv
+          . .venv/bin/activate
+          pip install .
+          pip install twine wheel setuptools
+          ./setup.py sdist bdist_wheel
+          pip install dist/trino-*.tar.gz
+          pip install dist/trino-*-py3-none-any.whl
+          twine check dist/trino-*-py3-none-any.whl dist/trino-*.tar.gz
+
+  github-release:
```

Comment:
None of the other Trino projects do releases on GitHub since there are no artifacts we'd want to provide via GitHub.

In this case the "GitHub release" will only contain the tarball of the source code, no actual artifacts and is hence redundant in my opinion. Maybe @electrum thinks otherwise?

Reply:
Since none of the other Trino projects do releases on GitHub, I dropped a whole job which was doing that.